### DISCUSSION TITLE
How to increase Endpoints count?

### DISCUSSION THREAD

**Discussion by SooryaMuthuraj** [apache/skywalking#11780] [Q&A]
Hi,

Can we increase the Endpoint count from 20 to 50 endpoints, as the clients application consists of more than 40 Endpoints. Is there any techniques to increase the endpoints detection?

**Comment by wu-sheng** [comment]
40 is not the limitation. You could search to get more detected.

**Reply by SooryaMuthuraj** [reply]
Unable to the see the endpoint performance details , after the limit gets reached.

![image](https://github.com/apache/skywalking/assets/154515976/0e4bb9a4-a1c4-45ed-adbc-61a1e91fdfb2)

**Reply by wu-sheng** [reply]
You could search, we wouldn't list all. That is impossible.

**Reply by vishalkadam18** [reply]
why this is impossible???

**Reply by wu-sheng** [reply]
Too many to list. Browser memory could overhead. Number of endpoint could be easiy over thousands, especially when you have parameters bundled restful style APIs.

**Comment by tanshanshan** [comment]
You can search for more endpoints using GraphQL queries.

**Reply by Seven-Wu-F** [reply]
Can you provide an example of a GraphQL query，ths

**Comment by wu-sheng** [comment]
We added this in 10.2 through https://github.com/apache/skywalking-booster-ui/pull/450

**Reply by SooryaMuthuraj** [reply]
Thanks for the update
